src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss

```diff
@@ -0,0 +1,256 @@
+.cdk-drag-preview {
+  box-sizing: border-box;
+  border-radius: 4px;
+  box-shadow: 0 5px 5px -3px rgba(0, 0, 0, 0.2), 0 8px 10px 1px rgba(0, 0, 0, 0.14), 0 3px 14px 2px rgba(0, 0, 0, 0.12);
+}
+
+.cdk-drag-placeholder {
+  opacity: 0;
+}
+
+.cdk-drag-animating {
+  transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
+}
+
+.example-box:last-child {
+  border: none;
+}
+
+.example-list.cdk-drop-list-dragging .example-box:not(.cdk-drag-placeholder) {
+  transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
+}
+
+.mat-header-cell {
+  background-color: var(--ubs-primary-grey);
+  color: var(--ubs-primary-white);
+  font-family: var(--primary-font);
+  border: 1px solid var(--ubs-primary-light-grey);
+  display: flex;
+  justify-content: center;
+  font-size: 14px;
+  width: 171px;
+  min-height: 39px !important;
+  min-width: 48px !important;
+  max-width: 300px !important;
+  overflow: hidden !important;
+  word-wrap: break-word !important;
+  white-space: nowrap !important;
+  overflow-wrap: break-word !important;
```

Comment:
https://css-tricks.com/when-using-important-is-the-right-choice/
well, this looks not like best practices